test/index-spec.js

```diff
@@ -0,0 +1,114 @@
+const should = require('should');
+const {
+  validCert,
+  validKey,
+  validBundle,
+  validBundleCert,
+  nonMatchingKey
+} = require('./__fixtures__/valid-ssl');
+
+const Validation = require('../lib/');
+
+describe('Validation', () => {
+  describe('#isValid functions should return false', () => {
+    it('#isValidCert', async () => {
+      (await Validation.isValidCert('')).should.be.false();
+    });
+    it('#isValidKey', async () => {
+      (await Validation.isValidKey('')).should.be.false();
+    });
+    it('#isValidCertKeyPair', async () => {
+      (await Validation.isValidCertKeyPair(validCert, '')).should.be.false();
+    })
+    it("#validateCertToDomain", async () => {
+      (await Validation.isValidCertToDomain(validCert, 'google.com')).should.be.false();
+    });
+    it('#isValidCertBundle', async () => {
+      (await Validation.isValidCertBundle(validBundle, '----NOT valid----')).should.be.false();
+    });
+  });
+  describe('#isValid function should return true', () => {
+   it('#isValidCert', async () => {
+      (await Validation.isValidCert(validCert)).should.be.true();
+    });
+    it('#isValidKey', async () => {
+      (await Validation.isValidKey(validKey)).should.be.true();
+    });
+    it('#isValidCertKeyPair', async () => {
+      (await Validation.isValidCertKeyPair(validCert, validKey)).should.be.true();
+    });
+    it("#validateCertToDomain", async () => {
+      (await Validation.isValidCertToDomain(validCert, 'mycustomguy.com')).should.be.true();
```

Comment:
I think you need a bunch more isValidCertToDomain tests, testing wildcards etc..